CHANGELOG.md

```diff
@@ -29,6 +29,111 @@ The changelog is generated during the release process using [ChangelogLinker](ht
+- [#1168] admin: product detail does not open in new window
+- [#1140] Registration of FriendlyUrlDataProviders is now done via service container
+- [#1044] remove unnecessary passing parameter as a reference, Thanks to [@pk16011990]
+- [#918] all entity factories now use EntityNameResolver
+
+#### Fixed
+
+- [#1248] DailyFeedCronModule set as lazy to avoid side-effects in constructor
+- [#1234] CountryFormType: removal of duplicate constraint
+- [#1232] LegalConditionsFacade::findArticle() returns null when the Article doesn't exist
+- [#1231] DomainController: fix non-existing route
+- [#1230] admin: consistency in separators in setting form
+- [#1227] AdministratorFacade: fix duplicate user name check
+- [#1226] admin: new order items have "Set prices manually" unchecked by default
+- [#1225] admin advanced search: reset button will not close the filter
+- [#1210] fixed ordering customers by telephone in administration
+- [#1204] dashboard statistics display zero trend when there are no data in previous week
+
+#### Removed
+
+- [#1097] end of support for helios-ag/fm-elfinder-bundle:^6.2.1
+
+### [shopsys/project-base]
+
+#### Changed
+
+- [#1256] enforce strict typing in project repository
+- [#1239] php-fpm dockerfile contains useful tools (vim, nano, mc, htop), Thanks to [@pk16011990]
+
+### [shopsys/shopsys]
+
+#### Added
+
+- [#1240] added Phing target that checks whether there is a project-base/vendor dir
+- [#1201] Start Building Your Application article
+- [#1245] acceptance test helper classes for graphical inputs
+- [#1181] all constants now have set their visibility
+- [#1096] use ElasticSearch to get data to readmodel
+- [#1004] support all languages and collations, Thanks to [@henzigo]
+- [#539] standards for .yaml and yaml.dist files, Thanks to [@sspooky13]
+
+#### Changed
+
+- [#1243] move admin assets into [shopsys/framework] repository
+- [#1192] commerceguys/intl is now upgraded to ^1.0.0
+- [#1176] BreadcrumbGenerators are now automatically registered in BreadcrumbResolver
+- [#1155] separated backend API from default project-base
+- [#1115] Product: availability cannot be set when using stock (and vice versa)
+- [#1123] entities: removal of dependencies on services
+- [#1072] Migration from IvoryCKEditorBundle to FOSCKEditorBundle
+- [#1047] encapsulation of collections inside of entities, Thanks to [@pk16011990]
+- [#933] phing target test-db-demo now exports products into elasticsearch with different index, Thanks to [@henzigo]
+
+#### Fixed
+
+- [#1219] fix grammar connected with "information" word
+- [#1183] fixed path for domain icons directory
+
+#### Removed
+
+- [#1193] remove deprecations from 7.x.x
+- [#1066] drop support for PHP 7.1
+
+### [shopsys/backend-api]
+
+#### Added
+
+- [#1055] API Product export
+- [#1203] create, update, delete products
+
+#### Changed
+
+- [#1208] backend-api installer is now updated with latest changes in build.xml file
```

Comment:
should be rather in "Fixed" section IMHO

Reply:
Fixed in c47a8ef99